libs/designsystem/slide/src/slides.component.ts

```diff
@@ -1,54 +1,111 @@
-import { EventEmitter } from '@angular/core';
 import {
-  ChangeDetectionStrategy,
+  AfterViewInit,
   Component,
   ContentChild,
+  EventEmitter,
   Input,
+  OnInit,
   Output,
   TemplateRef,
   ViewChild,
 } from '@angular/core';
-import { IonSlides } from '@ionic/angular';
-
+import { SwiperOptions } from 'swiper';
+import { register } from 'swiper/element/bundle';
+import { PlatformService, UniqueIdGenerator } from '@kirbydesign/designsystem/helpers';
 import { SlideDirective } from './slide.directive';
 
+// Swiper is not an Angular library,
+// so we need to use their web components and register them manually.
+// https://swiperjs.com/element
+register();
+
+export type SelectedSlide = {
+  slide: any;
+  index: number;
+};
+
+export type KirbySwiperOptions = SwiperOptions;
 @Component({
   selector: 'kirby-slides',
-  template: `
-    <ion-slides
-      [options]="slidesOptions"
-      #ionslides
-      (ionSlideDidChange)="onSlideChanged()"
-      ionSlidesDidLoad="slideTo(0)"
-    >
-      <ion-slide *ngFor="let slide of slides; let i = index">
-        <ng-container
-          *ngTemplateOutlet="slideTemplate; context: { $implicit: slide, index: i }"
-        ></ng-container>
-      </ion-slide>
-    </ion-slides>
-  `,
-  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './slides.component.html',
+  styleUrls: ['./slides.component.scss'],
 })
-export class SlidesComponent {
-  @ViewChild('ionslides', { static: false }) ionSlides: IonSlides;
-  @Input() slidesOptions: any;
-  @Input() slides: any[];
-  @Output() selectedSlide = new EventEmitter<any>();
+export class SlidesComponent implements OnInit, AfterViewInit {
+  constructor(private platform: PlatformService) {}
 
+  @ViewChild('swiperContainer') swiperContainer;
```

Comment:
Consider strong typing:
```suggestion
type SwiperContainer = HTMLElement & { initialize: () => void; swiper: Swiper };

export class SlidesComponent implements OnInit, AfterViewInit {
  constructor(private platform: PlatformService) {}

  @ViewChild('swiperContainer') swiperContainer: ElementRef<SwiperContainer>;
```